OpenIndiana doesn't have the Solaris identity:cert service which
deploys or creates the X.509 certificates for WebUI and RAD connections using the TLS transport.
Probably our ca-certificates service is an equivalent.

--- cups-2.4.2/scheduler/cups.xml.in.orig	2022-05-26 08:17:21.000000000 +0000
+++ cups-2.4.2/scheduler/cups.xml.in	2022-10-23 15:00:19.116296979 +0000
@@ -54,7 +54,7 @@
         grouping='require_all'
         restart_on='refresh'
         type='service'>
-        <service_fmri value='svc:/system/identity:cert'/>
+        <service_fmri value='svc:/system/ca-certificates'/>
     </dependency>
 
     <exec_method
@@ -78,7 +78,7 @@
             name='start'
             exec='@sbindir@/cupsd'
             timeout_seconds='60' >
-            <method_context>
+            <method_context security_flags="current">
                 <method_credential user='root' group='@CUPS_GROUP@' />
             </method_context>
         </exec_method>
@@ -165,7 +165,7 @@
             name='inetd_start'
             exec='@CUPS_SERVERBIN@/daemon/cups-lpd -o document-format=application/octet-stream'
             timeout_seconds='0'>
-            <method_context>
+            <method_context security_flags="current">
                 <method_credential user='@CUPS_USER@' group='@CUPS_GROUP@' />
             </method_context>
         </exec_method>
